src/containers/Form/components/FormNotes.jsx

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormHeader } from '@ndla/forms';
+import { colors, spacing } from '@ndla/core';
+import { injectT } from '@ndla/i18n';
+import styled, { css } from 'react-emotion';
+import { User, Time } from '@ndla/icons/common';
+import { Field } from '../../../components/Fields';
+import formatDate from '../../../util/formatDate';
+import {
+  resolveJsonOrRejectWithError,
+  fetchAuthorized,
+} from '../../../util/apiHelpers';
+import { NoteShape } from '../../../shapes';
+import Tag from '../../../components/Tag';
+
+const StyledFormNote = styled('div')`
+  display: flex;
+  width: 75%;
+  flex-direction: column;
+  border-bottom: 3px solid ${colors.brand.secondary};
+  padding: ${spacing.normal} 0;
+  &:last-child {
+    border: 0;
+  }
+`;
+
+const StyledFormNoteHeader = styled('div')`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const StyledFormNoteHeaderText = styled('span')`
+  color: ${colors.brand.grey};
+`;
+
+const iconStyle = css`
+  height: 20px;
+  width: 20px;
+`;
+
+class FormNotes extends React.Component {
+  constructor() {
+    super();
+    this.getUsername = this.getUsername.bind(this);
+    this.state = {
+      users: [],
+    };
+  }
+
+  async componentDidMount() {
+    const { notes } = this.props;
+    const userIds = notes.map(note => note.user);
+    const uniqueUserIds = Array.from(new Set(userIds)).join(',');
+    const users = await fetchAuthorized(
+      `/get_note_users?userIds=${uniqueUserIds}`,
+    ).then(resolveJsonOrRejectWithError);
```

Comment:
Bruk await i stedet for then. Og kanskje flytt til api-fil slik som andre steder?